internal/webserver/type/apache/enumeration/xpoweredbyheadergrab.go

```diff
@@ -0,0 +1,82 @@
+package webserver
+
+import (
+	"crypto/tls"
+	"net/http"
+	"strings"
+	"time"
+
+	webscan "github.com/Method-Security/webscan/generated/go"
+)
+
+type XPoweredByHeaderGrabLibrary struct{}
+
+func (XPoweredByHeaderGrabLib *XPoweredByHeaderGrabLibrary) ModuleRun(target string, config *webscan.WebServerTypeConfig) (*webscan.Attempt, []string) {
```

Comment:
Totally up to you, but from what I've seen in Go, typically the struct reference here (the `XPoweredByHeaderGrabLib`) is more often a less verbose, often single character reference (eg. `x`).

I personally do not mind the verbosity, but I would not have any objections to you following the go style with these.

Reply:
Interesting - Ya I made if verbose because I saw a future where we might have another header grab module so Im going to leave it but good to know for the future